src/Akka.Persistence.MongoDb/Akka.Persistence.MongoDb.csproj

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <PackageReference Include="Akka.Persistence.Query" Version="$(AkkaVersion)" />
     <PackageReference Include="akka.streams" Version="$(AkkaVersion)" />
+    <PackageReference Include="MongoDB.Driver" Version="2.11.4" />
```

Comment:
Why? We already have a reference to MongoDb.Driver on the line below

Reply:
Did not know how that happened.....I will take care of it.